aptos-move/block-executor/src/executor.rs

```diff
@@ -924,7 +901,7 @@ where
                 SchedulerTask::NoTask => scheduler.next_task(),
                 SchedulerTask::Done => {
                     drain_commit_queue();
-                    break;
+                    break Ok(());
```

Comment:
Is this a valid syntax?

Reply:
yup

Reply:
every block can return a value, including loops